src/components/Table/Table.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './styles.module.css';
+
+const Table = ({ header = [], body = [] }) => (
+    <div className={styles[`table-wrapper`]} style={{'--columns': header.length}}>
+        {header.map((item, i) => (
```

Comment:
Can you change `i` to `index`? Since most of our devs are pretty junior I think readability and semantics should be stressed as much as possible.

Reply:
Done in f370f14